.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "crates/wasi-crypto/spec"]
 	path = crates/wasi-crypto/spec
 	url = https://github.com/WebAssembly/wasi-crypto.git
+[submodule "crates/fuzzing/wasm-spec-interpreter/ocaml/spec"]
+	path = crates/fuzzing/wasm-spec-interpreter/ocaml/spec
+	url = https://github.com/WebAssembly/spec
```

Comment:
This submodule is 89MB. It will be checked out by cargo automatically when using Cranelift as git dependency like cg_clif does. Can you please make it lazily clone this repo only when fuzzing?

Reply:
I agree it's a bit large but I'm not sure exactly how to do that. Are you thinking of doing a `git clone ...` in the `build.rs` file for `wasm-spec-interpreter`? Isn't there a way to set `--depth 1` on the submodule somehow?

Reply:
From [this StackOverflow answer](https://stackoverflow.com/a/17692710) there is `git config -f .gitmodules submodule.<name>.shallow true`. I agree this would probably be preferable to a magic "git clone within the build script" solution.

If the shallow clone is still too large, another option could be to "rotate the tree", so to speak, and make this fuzzer plus the Wasm spec interpreter and submodule a root repo that pulls in wasmtime as a subrepo. It would make the oss-fuzz config a little more complex (cloning a separate repo to build this target) but would be workable. I'd still prefer it as you've done here over that solution if at all possible, though.

Reply:
I went with the shallow approach... the other options seem more complicated.